config/crd/bases/workload.nephio.org_amfdeployments.yaml

```diff
@@ -0,0 +1,366 @@
+                          type: string
+                      required:
+                      - address
+                      type: object
+                    name:
+                      description: Name defines the name of the interface
+                      maxLength: 253
+                      minLength: 1
+                      type: string
+                    vlanID:
+                      description: VlanID defines the specific vlan id associated
+                        on this interface
+                      type: integer
+                  required:
+                  - name
+                  type: object
+                type: array
+              networkInstances:
+                description: NetworkInstances defines the network instances associated
+                  with the NF deployment
+                items:
+                  description: A networkInstance is a Layer 3 forwarding construct
+                    such as a virtual routing and forwarding (VRF) instance,
+                  properties:
+                    bgp:
+                      description: bgp defines the BGP configuration associated with
+                        the network instance
+                      properties:
+                        autonomousSystem:
+                          description: ASNumber defines the AS number of the bgp process
+                          type: integer
+                        neighbors:
+                          description: BGPNeigbors defines the configuration of the
+                            BGP neighbor
+                          items:
+                            properties:
+                              Name:
+                                description: BGP interface name, MUST match the one
+                                  use in InterfaceConfig
+                                type: string
+                              address:
+                                description: Address defines the IPv4 or IPv6 address
+                                  of the BGP neighbor
+                                type: string
+                              peerAS:
+                                description: PeerAS defines the AS number of the bgp
+                                  peer
+                                type: integer
+                            required:
+                            - address
+                            - peerAS
+                            type: object
+                          type: array
+                        routerID:
+                          description: RouterID defines the router ID of the bgp process
+                          type: string
+                      required:
+                      - autonomousSystem
+                      - neighbors
+                      - routerID
+                      type: object
+                    dnns:
+                      description: dnns defines the dnns assocated with the network
```

Comment:
understand DNNs relevance to SMF and UPF, so is this required for free5gc AMF ?..assume that is the case ?

Reply:
yes it is. See orange configs